website/src/components/Submission/DataUploadForm.tsx

```diff
@@ -122,9 +126,12 @@ const InnerDataUploadForm = ({
     onError,
     group,
     referenceGenomeSequenceNames,
+    metadataTemplateFields,
 }: DataUploadFormProps) => {
-    const [metadataFile, setMetadataFile] = useState<File | null>(null);
-    const [sequenceFile, setSequenceFile] = useState<File | null>(null);
+    const [metadataFile, setMetadataFile] = useState<ProcessedFile | null>(null);
+    // can be null; if null -> don't apply mapping.
```

Comment:
```suggestion
    // The columnMapping can be null; if null -> don't apply mapping.
```

To avoid potential misunderstanding to which line this comment belongs to